string pilot_name
string crazyflie_name
int32 iteration_number
time init_time
pilot/Plan plan
pilot/Box fly_box
float64 fly_time